### PR TITLE
Patch Release 6.5.2 - update supported versions page

### PR DESCRIPTION
**Pull request by mgibson323**
## Description

This PR updates the supported versions page for 6.5.2

### CODE REVIEW

File: content/sensu-go/6.5/versions.md [hunk 1]
```diff
@@ -24,6 +24,7 @@ This table lists the supported versions of Sensu Go with links to active documen
 
 | Version | Release date     | Status    |     |
 | ------- |   -------------- | --------- | --- |
+6.5.2 | [October 28, 2021][74] | Supported | <a href="https://sensu-docs.s3.amazonaws.com/pdfs/sensu-go-6-5_sensu-docs.pdf"><img src="/images/download-icon.png" width="30" height="30" title="Download Offline Docs" alt="Download Offline Docs"></a>
```
Comment (hillaryfraley) [inline content/sensu-go/6.5/versions.md]:
```suggestion
6.5.2 | [October 28, 2021][75] | Supported | <a href="https://sensu-docs.s3.amazonaws.com/pdfs/sensu-go-6-5_sensu-docs.pdf"><img src="/images/download-icon.png" width="30" height="30" title="Download Offline Docs" alt="Download Offline Docs"></a>
```

Comment (hillaryfraley) [reply content/sensu-go/6.5/versions.md]: The link to add at the bottom of the page will be for the 6.5.2 release notes:

[75]: https://docs.sensu.io/sensu-go/latest/release-notes/#652-release-notes

Comment (mgibson323) [reply content/sensu-go/6.5/versions.md]: Thanks! I missed that at first but it should be in https://github.com/sensu/sensu-docs/pull/3450/commits/ac9303869e2988fa71f4b36862344bd87cb2e09c

**Review by hillaryfraley — APPROVED** [review]
👍

**Pull request merged**